en/assembly/vibration_isolation.md

```diff
@@ -0,0 +1,36 @@
+# Vibration Isolation
+
+This topic shows how to determine whether vibration levels are too high, and lists some simple steps to improve vibration performance.
+
+## Overview
+
+Flight Control boards with in-built accelerometers or gyros are sensitive to vibrations.
+High vibration levels can cause sensor clipping and sensor failures, resulting in a range of problems, from reduced flight efficiency/performance (leading to shorter flights and increased vehicle wear-and-tear), through to fly-aways and crashes.
+
+Airframes are designed to damp/reduce the amplitude of specific structural resonances at the autopilot mounting location.
```

Comment:
I wish all of them were! :)

Reply:
:-)

I changed this to "Well-designed airframes damp/reduce the amplitude of specific structural resonances at the autopilot mounting location."